### ISSUE TITLE
Details about reproduce the results of LKU-Net on OASIS dataset

### ISSUE THREAD

**Issue by AXu0511** [xi-jia/LKU-Net#5]
Hello, thank you very much for your outstanding work, I was recently reproducing your results. Did you train the half-resolution displacements directly or train the a full-resolution displacement first then downsamples the displacement? By the way, could you be so kind to share the OASIS version code?

**Comment by xi-jia** [comment]
We tried and updated both the full- and half-resolution code. The pre-trained half-resolution models are also released and can be found in the readme.

**Comment by rohitrango** [comment]
Hi there,

Thanks for the work. I tried re-training the neurite OASIS dataset, and the full-resolution training collapses after a few iterations. Here is what the training log looks like:

<img width="749" alt="image" src="https://github.com/xi-jia/LKU-Net/assets/15312514/0601accc-a7af-464c-bad2-1ee1d42e94cb">

I have used the default parameters provided in the file. Did you face any similar problems while training? Thanks in advance for the help.

**Comment by xi-jia** [comment]
@rohitrango, the corrected weights can be found in the command.txt, with --data_labda **1.0** (not used in code actually) --smth_labda  **0.01** (is different from the one in train.py) --mask__labda **1.0** (as default in the train.py)

loss = **1** * loss1 + **0.01** * loss2 +  **1.0** * loss3

**Comment by rohitrango** [comment]
Yes, I noticed that `data_labda` was not used, and I added it back (using 1.0) as of now.

I've changed `smth_labda` and `start_channels` appropriately from `command.txt`. I also wanted to use cross correlation instead of L2 loss (`using_l2` = 2), is that fine or does it make the training unstable? 

I've started training with the config you mentioned, fingers crossed!

**Comment by xi-jia** [comment]
@rohitrango
>  use cross correlation instead of L2 loss (using_l2 = 2), is that fine?

Yes. For NCC to achieve a satisfying Dice, the smth_labda should be much larger than 0.01 used for MSE. I vaguely remember we tuned a few sets of parameters like --data_labda 1.0 --smth_labda 0.1 --mask__labda 1.0 or --data_labda 1.0 --smth_labda 0.5 --mask__labda 1.0,  but the optimal Dice of NCC is 1%-2% lower than that of MSE for this specific dataset.

**Comment by rohitrango** [comment]
Sounds good. Thanks for the info.
